code/modules/admin/verbs/debug.dm

```diff
@@ -271,19 +271,32 @@ But you can call procs that are of type /mob/living/carbon/human/proc/ for that
 	message_admins(span_adminnotice("[key_name_admin(usr)] changed the outfit of [ADMIN_LOOKUPFLW(H)] to [dresscode]."))
 
 /client/proc/job_selector(mob/to_dress)
+	var/list/basejobs = list("Custom")
 	var/list/jobs = subtypesof(/datum/job)
 	var/list/selection = list()
 	for(var/datum/job/job as anything in jobs)
 		if(is_abstract(job))
 			continue
 		selection[job.title] = job
 
-	var/datum/job/selected = browser_input_list(src, "Select Job", "Job selection", selection)
+	var/datum/job/selected = browser_input_list(src, "Select Job", "Job selection", basejobs + selection)
 	if(!selected || QDELETED(src))
 		return
-	selected = SSjob.GetJobType(selection[selected])
-	if(!istype(selected))
-		return
+
+	var/datum/job/custom_job/selected_J
```

Comment:
custom_job seemingly has nothing unique about it therefore  there is not reason not to just change the previous selected var instead of making a new one

Reply:
yeah, changed